changelogs/71921-raise-bytes-for-binary-test

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - "Raise the number of bytes scanned by ansible-test to determine if a file is binary from 2048 to 4096 to accomodate the newest ansible-2.10.inv file (https://github.com/ansible/ansible/pull/71921)." 
```

Comment:
Changes to `ansible-test` are usually prefixed with `ansible-test - `. Since the previous value has changed twice recently I'd omit the original value. I'd also leave out the reference to the file which needed the change, since this change has a broader effect.

```suggestion
  - ansible-test - Raise the number of bytes scanned by ansible-test to determine if a file is binary to 4096. 
```

Reply:
Oh, and the file needs a `.yml` extension.

Reply:
ah, thanks